web/modules/custom/portland_openid_connect/portland_openid_connect.module

```diff
@@ -61,10 +61,11 @@ function portland_openid_connect_user_update($account)
  *    drush state:get epgov.user_sync.sync_now
  *    drush state:get epgov.user_sync.batch_size
  *    drush state:get epgov.user_sync.last_sync_date.DOMAIN
+ *    drush state:get epgov.user_sync.last_check_removals_date.DOMAIN
  *    drush state:get epgov.user_sync.resume_url.DOMAIN
+ *    drush state:get epgov.user_sync.drupal_user_offset.DOMAIN"
```

Comment:
Is the quote at the end of this line supposed to be there?

Reply:
Thanks for the catch. Fixed.